### PR TITLE
Add delay to SSRC removal from an RTCP BYE.

### PR DESCRIPTION
**Pull request by pthatcherg**
This is based on the feedback from Magnus.

### CODE REVIEW

**Comment by aboba** [comment]
LGTM.

**Comment by gloinul** [comment]
Yes, works for me.

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -6729,7 +6729,12 @@ a=msid:bbce3ba6-abfc-ac63-d00a-e15b286f8fce
           each SSRC indicated in the packet that is found in the
           incoming SSRC table, first deliver a copy of the packet to
           the "m=" line associated with that SSRC, but then remove the
-          entry for that SSRC from the incoming SSRC table.</t>
+          entry for that SSRC from the incoming SSRC table after an
+          appropriate delay (Similar to <xref target="RFC3550" />
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: I'm not a fan of normative language in parentheticals. Suggest rephrasing as:

    entry for that SSRC from the incoming SSRC table after an
    appropriate delay to account for "straggler packets", as 
    mentioned in the discussion of BYE handling in
    <xref target="RFC3550" />, Section 6.2.1.

**Comment by juberti** [comment]
LGTM with comment

File: draft-ietf-rtcweb-jsep.xml (hunk 1 above)
Comment (alvestrand) [inline draft-ietf-rtcweb-jsep.xml]: Should that be "as specified in 3550"? We're not adding new behavior here, we're reminding people what 3550 says on how to handle BYE.

Comment (fluffy) [inline draft-ietf-rtcweb-jsep.xml]: I think we should make change HTA proposes

**Comment by pthatcherg** [comment]
I have addressed the comments.  PTAL

**Comment by alvestrand** [comment]
LGTM

**Comment by pthatcherg** [comment]
FYI, I updated https://github.com/cdh4u/draft-sdp-bundle/pull/29 to match.

**Pull request merged**